wsdl/ver10/advancedsecurity/wsdl/advancedsecurity.wsdl

```diff
@@ -721,6 +721,11 @@
 						<xs:documentation>Indicates the maximum number of certification paths that the device can store simultaneously.</xs:documentation>
 					</xs:annotation>
 				</xs:attribute>
+				<xs:attribute name="SetCertPath" type="xs:boolean">
+					<xs:annotation>
+						<xs:documentation>Indicates support for modifying an existing certification path.</xs:documentation>
```

Comment:
Same comment as above.

Reply:
Agree with @bsriramprasad, this capability **SetCertPath** is for both **SetCertificationPath** and **SetCertPathValidationPolicy** methods, therefore better to clarify that in the annotation.